### PR TITLE
fix build errors

### PR DESCRIPTION
**Pull request by nusnewob**

### CODE REVIEW

File: config/nunjucks/filters.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const {
   isEmpty,
   isString,
   pickBy,
-  isNil,
+  isNil
```
Comment (marcofucci) [inline config/nunjucks/filters.js]: can you keep the comma?

File: package.json [hunk 2]
```diff
@@ -3,8 +3,11 @@
   "version": "1.0.0",
   "description": "The public front end for OMIS",
   "main": "src/app/app.js",
+  "engines": {
+    "node": "8.6.0"
+  },
   "scripts": {
-    "start": "NODE_ENV=production node --use-strict src/app/server.js",
+    "start": "npm run-script build && NODE_ENV=production node --use-strict src/app/server.js",
```
Comment (marcofucci) [inline package.json]: can we get rid of the `npm run build` here and add a `heroku-postbuild` with `npm run build` in it like in the datahub frontend? https://github.com/uktrade/data-hub-frontend/blob/develop/package.json#L119

Comment (nusnewob) [reply package.json]: yep, that's a much better idea, I'll change it.

File: config/nunjucks/filters.js (hunk 1 above)
Comment (nusnewob) [reply config/nunjucks/filters.js]: thought it was the comma caused build error, it actually was node version, I'll add it back.

**Pull request merged**